node-wasm/js/index.js

```diff
@@ -7,3 +7,4 @@ export async function spawnNode() {
     return client;
 }
 
+export * from "lumina-node-wasm";
```

Comment:
this apparently doesn't export the default export from lumina-node-wasm. If we want users to be able to just initialize wasm without using `spawnNode`, we must add
```suggestion
export * from "lumina-node-wasm";
export default init;
```

Reply:
Didn't know about it, changed!